lib/components/tooltip.js

```diff
@@ -127,6 +134,38 @@ const addNpmLink = (data) => {
   return npm;
 };
 
+const addAtomLink = (data) => {
+  if (!data || !data.name) {
+    return undefined;
+  }
+
+  const dirlink = new Directory(atom.workspace.getActiveTextEditor().buffer.file.getParent().path+"/node_modules/"+data.name)
+  const existDirlink = dirlink.existsSync()
+
+  const icon = el('i');
+  setAttr(icon, {
+    className: 'icon icon-file-directory atom-icon',
+  });
+
+  const iconWrapper = el('span', icon);
+  setAttr(iconWrapper, {
+    className: 'tab selected',
+  });
+
+  const atom_el = el('a', iconWrapper);
```

Comment:
Can we change the approach to `camelCase`? i.e. `atomEl` just for consistency sake 😄